qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/CreateBoxAccountUITest.java

```diff
@@ -73,13 +69,24 @@
 public class CreateBoxAccountUITest extends UIBoxHelper {
 
     @Rule
-    public ActivityTestRule<CreateAccountActivity> mActivityTestRule = new ActivityTestRule<>(CreateAccountActivity.class, false, true);
+    public ActivityTestRule<CreateAccountActivity> mActivityTestRule = new ActivityTestRule<CreateAccountActivity>(CreateAccountActivity.class, false, true){
```

Comment:
This works, but I like it better to just not launch the activity automatically but with testRule.launchActivity.